File: a059(perfect_square)/C++/perfect_square.cpp
#include <iostream>
#include <cmath>


int main(void) {
  int num_of_cases;
  while(std::cin >> num_of_cases) {
    int c[num_of_cases] = {0};
    for(int i = 0; i < num_of_cases; i++) {
      int lower_limit, upper_limit, sum = 0;
      std::cin >> lower_limit >> upper_limit;
      for(int number = lower_limit; number <= upper_limit; number++) {
        if (pow((int)sqrt(number), 2) == number) {
          sum += number;
        }
      }
      c[i] = sum;
    }
    for(int i = 0; i < num_of_cases; i++) {
      std::cout << "Case " << i+1 << ": " << c[i] << '\n';
    }
  }

  return 0;
}

File: a059(perfect_square)/Python/perfect_square.py


while True:
    try:
        num_of_cases = int(input())
        case = []
        for i in range(num_of_cases):
            lower_limit = int(input())
            upper_limit = int(input())
            sum = 0
            for number in range(lower_limit, upper_limit+1):
                if int(number ** 0.5) ** 2 == number:
                    sum += number
            case.append(sum)
        for i in range(num_of_cases):
            print(f'Case {i+1}: {case[i]}')
    except:
        break
